Story/Episode 3/np_010852.csv

```diff
@@ -65,16 +65,16 @@ text06007#0,"""Besides, there is a possibility<br>that this may occur to other<b
 text06008#0,"""I've seen the records of you<br>meeting【Persona】and analyzed<br>it accordingly."""
 text06009#0,"""He gave you a piece of advice,<br>but at the same time, it is<br>too vague."""
 text06010#0,"""Personally, I feel that we<br>should not ignore his words."""
-11_voice_NPC_5085_104#6,"""Sakuna-hime,the Ash Miko in Harkotan's legend."""
+11_voice_NPC_5085_104#6,"""Sukuna-hime,the Ash Miko in Harkotan's legend."""
 text07002#0,"""Although her title is a Miko,<br>she feels more like a god,<br>the god of Harkotan."""
 text07003#0,"""She doesn't seem keen to be<br>treated as a god, perhaps she<br>wants to be a treated as a normal<br>person. I can understand how<br>she feels..."""
 text07004#0,"""I do not wish to return to the<br>state where my consciousness was<br>in that pool of water. I want<br>to be like everyone else here."""
 text07005#0,"""Enough with the small talk...<br>Sukuna-hime is the god of good,<br>while Magatsu is a god of evil<br>and annihilation, the god that<br>harms the planet."""
 text07006#0,"""Be it Shironians or Kuronites,<br>it's destruction knows no bounds."""
 text07007#0,"""Sukuna-hime has sealed Magatsu<br>a long time ago but it seems<br>that the power of the seals are<br>fading."""
 text07008#0,"""While reapplying the seal,<br>【Double】's interference caused<br>a part of Magatsu to escape."""
-text07009#0,"""I am more concerned about how<br>【Double】is going to use Magatsu's<br>power than simply the fact that<br>a part of Magatsu escaped."""
-text07010#0,"""This isnt simply a problem for<br>Harkotan, but the universe as<br>well."""
+text07009#0,"""I am more concerned about how<br>【Double】is going to use Magatsus<br>power than simply the fact that<br>a part of Magatsu escaped."""
```

Comment:
Should this be "Magatsus's power", not "Magatsus Power"?

Reply:
Magatsu's and 【】need to be replaced with [ and ]